### DISCUSSION TITLE
Train Test Split

### DISCUSSION THREAD

**Discussion by Rishabh-Mer** [mrdbourke/pytorch-deep-learning#390] [Q&A]
Hi Daniel Bourke, 
Thank You so much for Deep Learning using PyTorch course

Currently, I am working on this dataset: https://www.kaggle.com/datasets/simolopes/wikiart-all-artpieces

This dataset contains more than 150K images, how do I split the data and label it based on artist images

Also, the final goal is to create a deep-learning model to classify artists based on images, Is this goal right? 

Thank You

**Comment by mrdbourke** [comment]
Hi @Rishabh-Mer,

It looks like the dataset file `wikiart_art_pieces.csv` contains the artist name in the `artist` column along with the image path.

You could group the files based on the artist name column and then move all the images with that certain artist name to a certain file.

This would separate the images into different categories (based on artists name).

If you're trying to classify paintings into who painted them (the artist name), this is the right approach.

**Reply by Rishabh-Mer** [reply]
Thank You,

data (```images```) for each class (```artist```) are imbalanced, what are the approaches to solve this imbalanced data
In your lecture you suggested, when we use ```CrossEntropyLoss()``` is has a weight parameter which can be used for imbalanced data
